examples/estimateBackground.py

```diff
@@ -38,7 +38,7 @@
 
 def getImage():
     imagePath = os.path.join(lsst.utils.getPackageDir("afwdata"),
-                             "DC3a-Sim", "sci", "v5-e0", "v5-e0-c011-a00.sci_img.fits")
+                             "DC3a-Sim", "sci", "v5-e0", "v5-e0-c011-a00.sci.fits")
```

Comment:
This change is needed because the file that was being loaded disappeared years ago (we used to persist MaskedImage and Exposure as three separate FITS files with suffixes `_img.fits, `_var.fits` and `_mask.fits`.

I will make this change (and the other one like it) as a separate commit. That should clear things up.